modules/dreamview/backend/sim_control/sim_control.cc

```diff
@@ -107,6 +98,57 @@ void SimControl::OnReceiveNavigationInfo(
   }
 }
 
+bool SimControl::InitStartPoint(double start_velocity,
+                                double start_acceleration) {
+  TrajectoryPoint point;
+  bool actual_position = false;
```

Comment:
acutal_position can just be replaced as start_point_set, making the state mutation easier to follow.